mustard_ui.py

```diff
@@ -1897,10 +1906,12 @@ def execute(self, context):
                         for j in range(len(mat.node_tree.nodes)):
                             if "MustardUI Float" in mat.node_tree.nodes[j].name and mat.node_tree.nodes[j].type=="VALUE":
                                 mustardui_add_option_item(obj.mustardui_additional_options, [mat.node_tree.nodes[j].name[len("MustardUI Float - "):], 'bpy.data.materials[\''+mat.name+'\'].node_tree.nodes[\''+mat.node_tree.nodes[j].name+'\'].outputs[0]', 'default_value', obj, 3])
+                            if "MustardUI BigFloat" in mat.node_tree.nodes[j].name and mat.node_tree.nodes[j].type=="VALUE":

```

Comment:
Not sure if this is needed, because we don't need to add this option to outfit additional properties, right?

Reply:
I imagined this would be useful for anything that is a material, whether it applies to skin, outfits, etc. I included this here because I am under the impression that this section allows for the float/bigfloat check to apply when checking for material settings on configured outfit properties.
The main use-case I have for this is displacement, normals, and spectacular. I find it is sometimes desirable to be able to adjust those above 1, usually to 2, 3 ,4, or 5.0

Reply:
To your comment above, "the additional options for the outfits part can be deleted, since those are already without limits." -  did not explicitly test that, I was under the assumption that they would have the same constraints, but I will test that now

Reply:
Alright, after some additional testing I am removing these couple lines, you are correct, they are unnecessary.